### PR TITLE
Add header_pages to README.md (see issue #52)

### PR DESCRIPTION
**Pull request by dirkgomez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,17 @@ To enable Google Anaytics, add the following lines to your Jekyll site:
 
 Google Analytics will only appear in production, i.e., `JEKYLL_ENV=production`
 
+### Link to specific files in header
```
Comment (adamvoss) [inline README.md]: Suggested title for consideration: `Customizing navigation links`

File: README.md [hunk 2]
```diff
@@ -132,6 +132,17 @@ To enable Google Anaytics, add the following lines to your Jekyll site:
...
+
+Add this to only link to the about and the linklists page in the default
+navigation header:
+
+```yaml
+header_pages:
+  - about.md
+  - linklists.md
+```
+
```
Comment (adamvoss) [inline README.md]: Suggested addition:

This allows you to set which pages you want to appear in the navigation area and configure order of the links.

File: README.md (hunk 1 above)
Comment (DirtyF) [inline README.md]: or something like `Defining the main navigation links`

File: README.md (hunk 2 above)
Comment (DirtyF) [inline README.md]: I agree with @adamvoss, it's best to share the reason why one may want to use this option.

File: README.md (hunk 1 above)
Comment (ashawley) [inline README.md]: Seems like there was a consensus on **Navigation links**  ;-)

Keep it minima!™

**Pull request closed without merging**